### ISSUE TITLE
class is a reserved word, consider using _class to replace class

### ISSUE THREAD

**Issue by geektutu** [stchris/untangle#53]
```xml
<?xml version="1.0"?>
<root>
    <class>Moc</class>>
</root>
```

```python
doc = untangle.parse("test.xml")
print(doc.root.class)

#     print(doc.root.class)
#                      ^
# SyntaxError: invalid syntax
```

class is a reserved word, consider using _class to replace class

**Comment by mhadam** [comment]
Agreed, ran into this issue where an element is named `None`.

**Comment by stchris** [comment]
Similar issue: #42 . Was fixed with #43
